AmplifyPlugins/DataStore/Sources/AWSDataStorePlugin/Storage/CascadeDeleteOperation.swift

```diff
@@ -143,6 +155,10 @@ public class CascadeDeleteOperation<M: Model>: AsynchronousOperation {
         
         let modelIds = queriedModels.map { $0.identifier(schema: self.modelSchema).stringValue }
 
+        if shouldLogDeveloperDefinedModel {
```

Comment:
I recommend creating a local function to replace these five `if` branches scattered throughout the code.

```swift
private func _log(
  _ message: @escaping @autoclosure () -> String,
  level log: (() -> String) -> Void
) { 
  guard shouldLogDeveloperDefinedModel else { return }
  log(message)
}
```

Then you can replace all of the
```swift
if shouldLogDeveloperDefinedModel { 
  self.log.<level>(...)
}
```

with
```swift
_log(..., level: log.<level>)
```

e.g.
```swift
// this
if shouldLogDeveloperDefinedModel {
  self.log.debug("[CascadeDelete.1] Deleting \(modelName) with identifiers: \(modelIds)")
}

// becomes this
_log("[CascadeDelete.1] Deleting \(modelName) with identifiers: \(modelIds)", level: log.debug)
```